enzyme/Enzyme/MLIR/Interfaces/GradientUtilsReverse.h

```diff
@@ -47,26 +42,8 @@ class MGradientUtilsReverse {
   IRMapping mapReverseModeBlocks;
   DenseMap<Block *, SmallVector<std::pair<Value, Value>>> mapBlockArguments;
 
-  IRMapping originalToNewFn;
-  std::map<Operation *, Operation *> originalToNewFnOps;
-
-  MTypeAnalysis &TA;
-
-  unsigned width;
-  ArrayRef<DIFFE_TYPE> ArgDiffeTypes;
-
   SymbolTableCollection &symbolTable;
 
-  mlir::Value getNewFromOriginal(const mlir::Value originst) const;
-  mlir::Block *getNewFromOriginal(mlir::Block *originst) const;
-  Operation *getNewFromOriginal(Operation *originst) const;
```

Comment:
Are these functions not used anywhere in `GradientUtilsReverse`?

Reply:
So they should use the versions defined in MGradientUtils (rather than forking their own copy)